central/main.go

```diff
@@ -395,10 +395,7 @@ func servicesToRegister(registry authproviders.Registry, authzTraceSink observe.
 		servicesToRegister = append(servicesToRegister, clusterCVEService.Singleton())
 		servicesToRegister = append(servicesToRegister, imageCVEService.Singleton())
 		servicesToRegister = append(servicesToRegister, nodeCVEService.Singleton())
-
-		if features.ObjectCollections.Enabled() {
-			servicesToRegister = append(servicesToRegister, collectionService.Singleton())
-		}
+		servicesToRegister = append(servicesToRegister, collectionService.Singleton())
```

Comment:
Put this behind postgres env var. Otherwise, this can panic because the singleton returns nil and we eventually call `RegisterServiceServer` for all registered services, in this call `RegisterServiceServer` on nil.

Reply:
Oh looks like git hides that line by default. [It is behind the flag in line 394](https://github.com/stackrox/stackrox/pull/4592/files#diff-43f1bb62accddc8c479ff1efa4084785789e26508ba52bc30096c6ca5555d070R394)

Reply:
Oh yes, had to hit expand